libraries/exoplayer_rtsp/src/main/java/androidx/media3/exoplayer/rtsp/reader/RtpH263Reader.java

```diff
@@ -168,7 +170,7 @@ public void seek(long nextRtpTimestamp, long timeUs) {
   private void getBufferFlagsAndResolutionFromVop(ParsableByteArray data, boolean gotResolution) {
     // Picture Segment Packets (RFC4629 Section 6.1).
     // Search for SHORT_VIDEO_START_MARKER (0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 1 0 0 0 0 0).
-    int currPosition = data.getPosition();
+    int currDataOffset = data.getPosition();
     long shortHeader = data.readUnsignedInt();
     if ((shortHeader & 0xffff) >> 10 == 0x20) {
```

Comment:
nit: shift and then mask, change `((shortHeader & 0xffff) >> 10 == 0x20)` to (1shortHeader >> 10) & 0xffff == 0x201.